deployed-mainnet.json

```diff
@@ -1,610 +0,0 @@
-{
```

Comment:
Not sure if we can remove deployed.json files yet

Reply:
Fixed in e0cac15